AUTHORS

```diff
@@ -8,3 +8,4 @@
 # Please keep the list sorted.
```

Comment:
should it be sorted lexicographical order? then Hengfei comes before Marty 😄

Reply:
Okay